docs/services/search.rst

```diff
@@ -25,6 +25,10 @@ only to document the methods it provides to its subclasses.
    :members:
    :show-inheritance:
 
+.. autoclass:: SearchQueryV1
```

Comment:
Within the docs I noticed were was an array for [deprecated items](https://github.com/globus/globus-sdk-python/blob/main/scripts/ensure_exports_are_documented.py#L25-L29). I wasn't sure if I needed to add `SearchQuery` to that array. Let me know if it does need to be added.

Reply:
That list of elements exists to explicitly exempt deprecated symbols from the documentation requirement.
It would not be appropriate to remove documentation for the `SearchQuery` class, since users may continue to use it up until we remove it in the 4.0.0 release.